run/system-package/package.json

```diff
@@ -0,0 +1,24 @@
+{
+  "name": "graphviz-web",
+  "version": "1.0.0",
+  "description": "Demonstrates a Cloud Run service which provides a CLI tool over HTTP.",
+  "main": "index.js",
+  "private": true,
+  "repository": {
```

Comment:
I don't this this property is needed in this `package.json`.

Reply:
I got an error for not having it, so added it. Maybe it's linked to having private: true?